### PR TITLE
2020 summer meeting

### PR DESCRIPTION
**Pull request by jiwoncpark**
I incorporated feedback from the SOC on the structure and wording. Please review, in particular, how I implemented the indentation for the question regarding volunteer responsibilities! (I got feedback that that question should look like a follow-up question to the checkbox question right before it.) There is also a long multi-select checkbox for all of the WG sessions.

### CODE REVIEW

File: registration_server.py [hunk 1]
```diff
@@ -18,15 +18,33 @@ class Participant(db.Model):
...
-    lname = db.Column(db.String(100))
-    sname = db.Column(db.String(100))
-    pronoun = db.Column(db.String(100))
+    #lname = db.Column(db.String(100))
+    #sname = db.Column(db.String(100))
     contact = db.Column(db.String(5))
     volunteer = db.Column(db.String(5))
     volunteer_responsibilities = db.Column(db.String(200))
     # visa    = db.Column(db.String(5))
+    CL = db.Column(db.String(5))
```
Comment (EiffL) [inline registration_server.py:30]: If you are not usisng  these fields anymore, better to remove them

File: templates/success.html [hunk 2]
```diff
@@ -3,23 +3,22 @@
...
 <div class="container">
-  <h2>You are registered for the DESC meeting !</h2>
+  <h2>You are registered for the DESC meeting!</h2>
   <br>
   <br>
-  {{data.first_name}}, we are looking forward to seeing at our virtual meeting.
+  {{data.first_name}}, we are looking forward to seeing you at our virtual meeting.
   <br>
-  Please check your emails for confirmation of your payment.
+  Please check your emails for confirmation of your registration.
```
Comment (EiffL) [inline templates/success.html:19]: Actually.... We don't send confirmation emails, do we? I used to happen automatically as a by-product of the payment, but we have no paymenet

File: registration_server.py (hunk 1 above)
Comment (jiwoncpark) [reply registration_server.py:30]: Right, thank you!

File: templates/success.html (hunk 2 above)
Comment (jiwoncpark) [reply templates/success.html:19]: Ahh. Is it possible to send confirmations emails? This feature was requested by SOC. I wonder if the implementation will be tricky?

Comment (EiffL) [reply templates/success.html:19]: hummmmmmmmmmmmmmmmmm It's doable of course, but not trivial :-/ most likely we would need to use an additional service

Comment (EiffL) [reply templates/success.html:19]: Ok, let's leave that message for now, and we just need to find out how to send an email from the `registration_server.py`

Comment (jiwoncpark) [reply templates/success.html:19]: Then I think it's fine... they can check their names on Confluence anyway.

**Pull request merged**